src/gatsby-node.js

```diff
@@ -1,6 +1,46 @@
 const fetch = require("node-fetch")
 const queryString = require("query-string")
 
+exports.createSchemaCustomization = ({ actions }) => {
```

Comment:
Thanks for the PR!

As mentioned in the [Gatsby docs](https://www.gatsbyjs.org/docs/schema-customization/):

> Actions to customize Gatsby’s schema generation are made available in the createSchemaCustomization (available in Gatsby v2.12 and above), and sourcesNodes APIs.

I would then suggest mentioning that the Gatsby Source Meetup plugin requires `gatsby >= 2.12.0` in the README and bump the peerDependency requirement in the `package.json`:

https://github.com/phacks/gatsby-source-meetup/blob/e7660c6cc6c4748f488c8d30725ed775d58c2406/package.json#L34-L36

Reply:
Thanks @phacks .
Suggestion applied.